src/Chirp.Web/Pages/Shared/_Layout.cshtml

```diff
@@ -11,7 +11,7 @@
     <div class=page>
         <h1><img src="/images/icon1.png" alt="Icon1" />Chirp!</h1>
         <div class=navigation>
-            @if (User.Identity.IsAuthenticated)
+            @if (User?.Identity.IsAuthenticated == true)
```

Comment:
The == true is unnecessary

Reply:
in order to eliminate warnings, i added the user null check "?" which makes the "== true" necessary as the value can be: true, false or null

Reply:
<img width="730" alt="Skærmbillede 2023-11-18 kl  18 46 32" src="https://github.com/ITU-BDSA23-GROUP3/Chirp/assets/133771163/8ce7f4ec-a387-42a0-aaa2-0909a7e7df74">